src/hls-adapter.js

```diff
@@ -136,6 +136,12 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
     if (Utils.Object.hasPropertyPath(config, 'playback.useNativeTextTrack')) {
       adapterConfig.subtitleDisplay = Utils.Object.getPropertyPath(config, 'playback.useNativeTextTrack');
     }
+    if (Utils.Object.hasPropertyPath(config, 'playback.recoverDecodingErrorDelay')) {
```

Comment:
remove this config, it doesn't exist on player or adapter

Reply:
and enable passing the fps settings:
fpsDroppedMonitoringPeriod and fpsDroppedMonitoringThreshold and enable flag capLevelOnFPSDrop.
All under a config called `abr`

Reply:
fixed, changed according to a new config